### PR TITLE
HERETICS : fixes an infinite summons exploit.

### PR DESCRIPTION
**Pull request by Djiq**
## About The Pull Request

There was an exploit caused by delaying a proc caused by PollGhosts(). Since the inital proc never cleaned up the items you could spawn as many heretics summons as fast as you could click. 

## Why It's Good For The Game

:agony:

## Changelog
:cl:
fix: You can no longer infinitely spawn heretics summons
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/eldritch_demons.dm [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	speed = 0
 	a_intent = INTENT_HARM
 	stop_automated_movement = 1
-	AIStatus = AI_ON
+	AIStatus = AI_OFF
```
Comment (ATH1909) [inline code/modules/mob/living/simple_animal/eldritch_demons.dm:19]: 👀

Comment (EdgeLordExe) [reply code/modules/mob/living/simple_animal/eldritch_demons.dm:19]: intended, the way summons work would cause the monster to start hitting random stuff.

**Review by kriskog — DISMISSED** [review]
Checks are failing.
```
code/modules/antagonists/eldritch_cult/eldritch_effects.dm, line 46, column 20:

error: undefined var: "atoms_in_use"
```

**Comment by EdgeLordExe** [comment]
E

File: code/modules/mob/living/simple_animal/eldritch_demons.dm [hunk 2]
```diff
@@ -37,8 +37,6 @@
 /mob/living/simple_animal/hostile/eldritch/Initialize()
 	. = ..()
 	add_spells()
-	//by default
-	mind.add_antag_datum(/datum/antagonist/heretic_monster)
```
Comment (kriskog) [inline code/modules/mob/living/simple_animal/eldritch_demons.dm:41]: Not adding this anymore?

Comment (EdgeLordExe) [reply code/modules/mob/living/simple_animal/eldritch_demons.dm:41]: Mind isnt added on Initalize, so it just runtimes every time.

Comment (kriskog) [reply code/modules/mob/living/simple_animal/eldritch_demons.dm:41]: So is it added after someone takes control of it?

Comment (EdgeLordExe) [reply code/modules/mob/living/simple_animal/eldritch_demons.dm:41]: yes

**Pull request merged**